.github/workflows/run-tests.yml

```diff
@@ -31,11 +31,23 @@ jobs:
         python3 --version
         python3 -c "import multiprocessing as m; print('CPUs:', m.cpu_count())"
 
-    - name: Install dependencies
+    - name: Install dependencies (Ubuntu)
+      if: startsWith(matrix.os, 'ubuntu')
       run: |
         sudo apt-get -y update
         sudo apt-get -y install -qq gir1.2-gdkpixbuf-2.0 gir1.2-glib-2.0 gir1.2-gtk-3.0 gir1.2-pango-1.0 gir1.2-webkit2-4.0 python3 python3-enchant python3-gi python3-yaml tox
 
+    - name: Install dependencies (macOS)
+      if: startsWith(matrix.os, 'macos')
+      run: |
+        brew install gtk
+        brew install pygobject3 glib libffi cairo gtksourceview3
+        brew install atk librsvg
+        brew install gsettings-desktop-schemas adwaita-icon-theme
+        brew install enchant
```

Comment:
Can we combine this into one brew command, i.e., on one line?

Reply:
I've pushed another commit to check this. Let's see! 🤞

Reply:
It seems to be working! \o/